.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Also tell us, what did you expect to happen? If applicable, add screenshots to help explain your problem.
```

it asks what we expected to happen twice which is a bit redundant, we can remove the middle bit
```suggestion
      label: What happened? If applicable, add screenshots to help explain your problem.
```